packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -198,6 +200,7 @@ exports.question = async (req, res, journeyId) => {
 		listLink: journey.baseUrl,
 		navigation: ['', backLink],
 		layoutTemplate: hasJourneyTemplate,
+		pageCaption: sectionObj?.name,
```

Comment:
It looks like the component views already write out a caption if a value of `caption` rather than `pageCaption` is passed in so, for consistency, we should either just use `caption` and add something to the summary list view to show a caption, or we should use `pageCaption` and remove the captions from the individual views.